src/prefect/agent/docker/agent.py

```diff
@@ -419,11 +443,18 @@ def deploy_flow(self, flow_run: GraphQLResult) -> str:
             networking_config=networking_config,
             labels=labels,
         )
-
+        # Connect the rest of the networks
+        if self.networks:
+            for network in self.networks[1:]:
+                self.docker_client.connect_container_to_network(
+                    container=container, net_id=network
+                )
         # Start the container
         self.logger.debug(
             "Starting Docker container with ID {}".format(container.get("Id"))
         )
+        if self.networks:
+            self.logger.debug(f"Adding container to docker networks: {self.networks}")
```

Comment:
Could you also log the container ID here in this debug statement?

Reply:
Updated to also log the container ID.